explorer/graphql/data/message_provider.go

```diff
@@ -100,7 +100,7 @@ type dbXMsgRow struct {
 	ReceiptID             sql.NullInt64
 	ReceiptTxHash         *common.Hash
 	ReceiptSuccess        sql.NullBool
-	ReceiptRelayerAddress common.Address
+	ReceiptRelayerAddress *common.Address
```

Comment:
All other properties of the receipts are `sqlNull*` or a pointer. This would fail with an sql driver error:
```
scanning row: sql: Scan error on column index 15, name \"receipt_relayer_address\": can't scan <nil> into Address
```
if the receipt doesn't exist (i.e. for pending XMsgs).

Reply:
I'll add it to the e2e tests in a followup PR.